scripts/hail_batch/snp_chip_generate_pca/snp_chip_generate_pca.py

```diff
@@ -0,0 +1,47 @@
+"""
+Generate PCA on combined TOB-WGS/SNP-chip data.
+"""
+
+import click
+import hail as hl
+import pandas as pd
+
+TOB_WGS = 'gs://cpg-tob-wgs-main/mt/v2-raw.mt/'
```

Comment:
Don't we have a newer version with more samples?

Reply:
Yes, we do! Thanks for pointing me to this. I actually had no idea there was a new one.